src/AbstractChart.tsx

```diff
@@ -97,6 +97,30 @@ class AbstractChart<
     };
   }
 
+  getPropsForVerticalLabels() {
+    const {
+      propsForVerticalLabels = {},
+      color,
+      labelColor = color
+    } = this.props.chartConfig;
+    return {
+      fill: labelColor(0.8),
```

Comment:
where does `labelColor(0.8)` come from? is this a new addition in this diff? why is it necessary?

Reply:
> where does `labelColor(0.8)` come from? is this a new addition in this diff? why is it necessary?

https://github.com/indiespirit/react-native-chart-kit/blob/a499f7e212daf8e8f15286985e9007fcc0147818/src/AbstractChart.tsx#L95
Respect to the original `getPropsForLabels()`, looks good to me.

Reply:
Yes i havent changed the original props